.pylintrc

```diff
@@ -76,7 +76,6 @@ enable=abstract-class-instantiated,
        assert-on-tuple,
        assignment-from-no-return,
        assignment-from-none,
-       bad-builtin,
```

Comment:
Why is this removed?

Reply:
Because in the `obj_size` function it uses `map` built-in on one line and is 50% faster than using a generator expression - which is pretty important for such a heavily used function.